A41-xds-rbac.md

```diff
@@ -32,13 +32,12 @@ Envoy defines the [RBAC HTTP filter][RBAC filter] that is well-suited for
 limiting which nodes can access which services within a mesh as encouraged by
 the principle of least privilege.
 
-The resource name in REST is present in the HTTP path so technically RBAC could
-be used as a precise service-level resource authz engine with REST. However,
-this 1) requires another filter to perform end-user authentication as TLS just
-provides service-level authn and 2) does not scale as RBAC provides no mechanism
-for loading rules on-demand. gRPC encodes service-level resource names (if they
-exist) in the request message which RBAC cannot access, but this is not seen as
-a major loss as RBAC is poorly suited for service-level resource authz.
+RBAC is not well-suited for precise service-level resource authorization, even
```

Comment:
Thanks, I understand this paragraph a lot better now.

I find the phrase "service-level resource authorization" a little confusing; maybe change that to just "resource-level authorization", so that the distinction between service-level and resource-level is clearer?

Or, maybe even better, how about this wording:

"""
RBAC is well-suited for service-level authorization, but it is not well-suited for resource-level authorization of the kind usually done in REST, even if the resource name is exposed to the engine.  Resource-level authorization 1) requires another filter to perform end-user authentication as TLS just provides service-level authn and 2) does not scale as RBAC provides no mechanism for loading rules on-demand.
"""

Reply:
Note: I still need to muck with this. I've got a concern about the replacements you offered so need to look at it a bit. The other changes were small so I did those quickly, but I'll come back around to this.

Reply:
Soo... I made a change similar to what you suggested, and thought I was happy and then re-read it one too many times and hated it. So instead, I've reworked this paragraph again. Essentially, I now spend a sentence describing this "per-resource authz" stuff from a service's perspective to provide enough context that the words are (hopefully) less ambiguous.